playbooks/support/on-call-slo.md

```diff
@@ -0,0 +1,13 @@
+# On-Call SLO
+
+This document describes a "Service-Level Objective" for the hours during with engineers are expected to be monitoring for incidents during their on-call schedule.
+
+![visualization of our on-call SLO](images/on-call-slo.jpg)
+
+## What we promise
+- Work days (excludes holidays)
```

Comment:
Thoughts on linking to the [official People Ops policy doc](https://docs.google.com/document/d/16FQ0nKIJ_Eb11kWQd70neB7V3i51ixUsf-CB5f_rAMo/edit#heading=h.d93f947ory) for ease of reference when determining if a certain day is a holiday?